docs/NEWSLOG.md

```diff
@@ -1,5 +1,12 @@
+## August 15, 2020
+## Auto-backup (ALPHA)
+Have openHABian automatically take daily syncs of your internal SD card to
```

Comment:
openHABian can automatically take daily backup syncs of your internal SD card to another device in an external port. This allows for fast swapping of cards to reduce impact of a failed SD card. The external device will also be used to setup an Amanda backup system.

Reply:
ok